src/features/wallet/Wallet.tsx

```diff
@@ -0,0 +1,263 @@
+                onClick={() => {
+                  disconnect();
+                }}
+                variant="secondary"
+              >
+                Disconnect
+              </Button>
+            )}
+          </div>
+
+          <div tw="my-5 mx-3 flex-grow border-r">
+            <p className="sm-title">UMA Balance</p>
+            <div className="value-tokens">
+              <span>{formatWalletBalance(umaBalance)[0]}</span>
+              <span>{formatWalletBalance(umaBalance)[1]}</span>
+            </div>
+            <p className="value-dollars">
+              $
+              {umaBalance && umaPrice
+                ? calculateUMATotalValue(
+                    umaPrice.market_data.current_price.usd,
+                    umaBalance
+                  )
+                : "$00.00"}
+              USD
+            </p>
+          </div>
+          <div tw="my-5 mx-3 pl-5 flex-grow border-r">
+            <p className="sm-title">Total UMA Collected</p>
+            <div className="value-tokens">
+              <span>{formatWalletBalance(totalUmaCollected)[0]}</span>
+              <span>{formatWalletBalance(totalUmaCollected)[1]}</span>
+            </div>
+            <p className="value-dollars">$00.00 USD</p>
+          </div>
+          <div tw="my-5 mx-3 pl-5 flex-grow">
+            <p className="sm-title">Available Rewards</p>
+            <div className="value-tokens">
+              <span>{formatWalletBalance(availableRewards)[0]}</span>
+              <span>{formatWalletBalance(availableRewards)[1]}</span>
+            </div>
+            <p className="value-dollars">$00.00 USD</p>
+          </div>
+          <div tw="py-10 pl-5 ml-auto flex-none">
+            <Settings onClick={() => open()} tw="cursor-pointer" />
+          </div>
+        </div>
+        <Modal isOpen={isOpen} onClose={close} ref={modalRef}>
+          <StyledModal>
+            <h3 className="header">Two Key Voting</h3>
+            <p tw="opacity-50 mb-4 text-center">
+              You are not currently using a two key voting system. To deploy
+              one, provide your cold key address. Click here to learn more about
+              the two key voting system.
+            </p>
+            <div tw="flex items-stretch">
+              <Disconnected tw="flex-grow">Not Connected</Disconnected>
+              <div className="open-form" tw="flex-grow text-right">
+                Add Cold Wallet Address
+              </div>
+            </div>
+          </StyledModal>
+        </Modal>
+      </div>
+    </StyledWallet>
+  );
+};
+
+const StyledWallet = styled.div`
+  background-color: #f5f5f5;
+
+  ${tw`max-w-full pt-5 mt-5 pb-1`};
+  .wrapper {
+    ${tw`max-w-7xl mx-auto py-5 px-8 my-10`}
+    background-color: #fff;
+  }
+  .wallet-title {
+    font-weight: 600;
+    font-size: 1.5rem;
+  }
+  .sm-title {
+    color: #000;
+    margin: 0 0 12px;
+    font-weight: 400;
+    opacity: 0.5;
+  }
+  .value-tokens {
```

Comment:
same minor comments, classes can give specificity issues, as its easy to use the same class name in a parent and then the parent selector would have the same specificity (or more depending on the circumstances)

Reply:
Let me test this out.

Reply:
I think I might have a workaround that's not too intrusive.

It would mean you could still add stylings above, but could easily override them locally.